services/minespace-web/src/styles/components/CommonPageHeader.scss

```diff
@@ -48,4 +48,8 @@
         background-color: $pure-white;
 
     }
+
+    .common-page-header {
+        height: calc(65px + $header-height);
```

Comment:
I am wondering if it was necessary to set the height on common-page-header and the `top` property on `permit-status-banner`- it looks like those properties are already correct on test (on the permit conditions page and on the project overview page). Just thinking about how fiddly those can be to maintain in coordination with the ant-tabs ones.

Reply:
When I reverted the styles in the fixed-tab-container there were a lot of trickle down styles that messed up the positioning of the top banner in permits.
You're correct that these are annoyingly fiddly to maintain, but I think these are all necessary unless we separate out some of the other styling from the fixed-tab section which would also have to be updated on Core.